repos/fdbt-site/src/pages/api/selectExports.ts

```diff
@@ -40,7 +42,8 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
     const links = products.map((product) => product.matchingJsonLink);
 
     await triggerExport({ noc, paths: links, exportPrefix: exportName });
-
-    redirectTo(res, '/products/exports');
+    const currTime = new Date().getTime() / 1000;
```

Comment:
I dont understand how this works - new Date().getTime() returns 1677164983380 , why do we divide that by 1000?

Reply:
to get the time in seconds

Reply:
Makes sense - but it shouldnt be in this API.